core/jvmMain/src/kotlinx/io/buffer/JvmByteOrder.kt

```diff
@@ -2,13 +2,12 @@
 
 package kotlinx.io.buffer
 
-actual enum class ByteOrder(val nioOrder: java.nio.ByteOrder) {
-    BIG_ENDIAN(java.nio.ByteOrder.BIG_ENDIAN),
-    LITTLE_ENDIAN(java.nio.ByteOrder.LITTLE_ENDIAN);
+public actual enum class ByteOrder  {
```

Comment:
In old IO it is not a type-alias.
We cannot make it a type-alias because BO in `java.nio` is class with private ctor and three static members. This is something I'd like to avoid in Kotlin IO